### PR TITLE
[MIG] pr_line_related_po_line: migrate to 16.0 Task#66187

### PR DESCRIPTION
**Pull request by Hernannc00000001**
[Task# 66187](https://www.vauxoo.com/web#action=464&active_id=222&cids=1&id=66187&menu_id=1490&model=project.task&view_type=form)

### CODE REVIEW

**Comment by Hernannc00000001** [comment]
@luisg123v

**Review by luisg123v — CHANGES_REQUESTED** [review]
Hi @Hernannc,

I see you  brought commit history correctly. However, there are a few issues in the migration:
- The migration commit (f9079ca6) message and author is a copy of the previous commit, it should be:
`[MIG] pr_line_related_po_line: migrate to 16.0`
- Also edit PR title to match the above commit title, adding task at the end
- You're uploading binary files like `pr_line_related_po_line/models/__pycache__/purchase_requisition_line.cpython-310.pyc`
- Please re-export translation file `es.po`

**Comment by luisg123v** [comment]
In addition, task is wrong, it should be [T#66187](https://www.vauxoo.com/web#action=464&active_id=222&cids=1%2C12%2C14%2C16%2C17%2C18&id=66187&menu_id=1490&model=project.task&view_type=form), as I specified in [the related issue](https://git.vauxoo.com/vauxoo/sbd/-/issues/122)

**Comment by Hernannc00000001** [comment]
@luisg123v improvements uploaded, check please

File: pr_line_related_po_line/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+#. module: pr_line_related_po_line
+#: model:ir.model,name:pr_line_related_po_line.model_purchase_requisition_line
+#: model:ir.model.fields,field_description:pr_line_related_po_line.field_purchase_order_line__purchase_requisition_line_id
+msgid "Purchase Requisition Line"
+msgstr "Línea solicitud de compra"
+
+#. module: pr_line_related_po_line
+#: model:ir.model.fields,field_description:pr_line_related_po_line.field_purchase_order_line__display_name
+#: model:ir.model.fields,field_description:pr_line_related_po_line.field_purchase_requisition_line__display_name
+msgid "Display Name"
```
Comment (luisg123v) [inline pr_line_related_po_line/i18n/es.po]: Are you sure this is the correct position?

Terms are sorted alphabetically when re-exporting the file.

Is this the order the file was generated?

**Review by luisg123v — CHANGES_REQUESTED** [review]

**Comment by luisg123v** [comment]
@Hernannc,

Regarding [this comment](https://github.com/Vauxoo/addons-vauxoo/pull/1594#pullrequestreview-1324117209):
- Pr title was not updated, you added task ID in commit title instead
- Now there are two commits, you can run `git log` and you will see them
- Missing blank line between title and description
- Description shouldn't be indented
- Description says you're removing pyc files, but you shouldn't add them in the first place
- Description says you're adding `.gitignore`, but not needed anymore because it's added in CI initialization. Besides, even if you would need to, it should be in a separated commit, as it's not part of the migration

File: pr_line_related_po_line/i18n/es.po (hunk 1 above)
Comment (luisg123v) [reply pr_line_related_po_line/i18n/es.po]: Neither replied nor applied

**Review by luisg123v — CHANGES_REQUESTED** [review]
The following point is not applied yet:

> Description shouldn't be indented

**Comment by luisg123v** [comment]
Besides, there still are two extra commits

**Comment by luisg123v** [comment]
@Hernannc I've fixed title by you.

File: pr_line_related_po_line/i18n/es.po (hunk 1 above)
Comment (Hernannc00000001) [reply pr_line_related_po_line/i18n/es.po]: fixed   [here](https://github.com/Vauxoo/addons-vauxoo/pull/1594/files#diff-50a8e001fe03eefa20cc8023cc12b96f2f6494756e567db67dfdd8e4a7a969b5)

**Comment by Hernannc00000001** [comment]
@luisg123v updated

**Review by luisg123v — APPROVED** [review]
LGTM :+1:

**Pull request merged**